med-inventory/src/layout/NavBar.jsx

```diff
@@ -10,6 +10,7 @@ import Typography from "@material-ui/core/Typography";
 import Divider from "@material-ui/core/Divider";
 import IconButton from "@material-ui/core/IconButton";
 import MenuIcon from "@material-ui/icons/Menu";
+import HelpIcon from "@material-ui/icons/Help";
```

Comment:
Mention all the changes done in description. Keeping commits atomic will help things in long run. 
1. Logo Change.
2. Help Icon.
3. Other change....

Reply:
Going forward I will mention all the changes done